client/main.go

```diff
@@ -101,22 +58,13 @@ func NewClientConn(opt *ConnOptions) (*grpc.ClientConn, error) {
 
 // GetConnection returns a gRPC client connection.
 func GetConnection() (*grpc.ClientConn, error) {
-	opts := ConnOptions{
-		CertURL:       env.Get("TINKERBELL_CERT_URL"),
-		GRPCAuthority: env.Get("TINKERBELL_GRPC_AUTHORITY"),
-		TLS:           env.Bool("TINKERBELL_TLS", true),
-	}
-
-	if opts.GRPCAuthority == "" {
+	authority := env.Get("TINKERBELL_GRPC_AUTHORITY")
+	if authority == "" {
 		return nil, errors.New("undefined TINKERBELL_GRPC_AUTHORITY")
 	}
 
-	if opts.TLS {
-		if opts.CertURL == "" {
-			return nil, errors.New("undefined TINKERBELL_CERT_URL")
-		}
-	}
-	return NewClientConn(&opts)
+	tls := env.Bool("TINKERBELL_TLS", true)
+	return NewClientConn(authority, tls)
```

Comment:
nit: could inline this and make it a const?
```suggestion
	return NewClientConn(authority, env.Bool(ENV_TINKERBELL_TLS, true))
```